src/Test.UnitTests.Sarif.Multitool.Library/MergeCommandTests.cs

```diff
@@ -94,7 +101,7 @@ private void PrepareFileSystemMock(string inputResourceName, string inputFolderP
                 return;
             }
 
-            if (inputResourceName.EndsWith("DuplicatedResults.sarif"))
+            if (inputResourceName.EndsWith("DuplicatedResults.sarif") || inputResourceName.EndsWith("FileNameOnlyWithoutPath.sarif"))
```

Comment:
>inputResourceName [](http://example.com/codeflow?start=73&length=17)

drop down to a second line, add Ordinal comparison to both.

Reply:
fixed